src/pages/DetailPage/components/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,14 @@
+import '../ProductDetail/ProductDetail.scss';
+
+function ProductDetail() {
+  return (
+    <div className="productDetail">
+      <img src="./images/1.jpg" alt="detail" />
+      <img src="./images/2.jpg" alt="detail" />
+      <img src="./images/3.jpg" alt="detail" />
+      <img src="./images/4.jpg" alt="detail" />
+    </div>
+  );
+}
+
```

Comment:
```suggestion
    <div className="productDetail">
      {DETAIL_IMAGES. map(({id,src,alt})=><img key={id} src={src} alt={alt}/>)}
    </div>
  );
}
const DETAIL_IMAGES = [
{id:1, src:"./images/1.jpg", alt:"detail"},
{id:2, src:"./images/1.jpg", alt:"detail"},
{id:3, src:"./images/1.jpg", alt:"detail"},
{id:4, src:"./images/1.jpg", alt:"detail"},
]
```

이렇게 상수데이터로 관리하면 추후 데이터 추가,삭제,수정에 용이합니다